### PR TITLE
Decsribe role of chairs more

### LINKED ISSUE #45: Remove "well actually" example
As per the thread beginning https://github.com/w3c/PWETF/pull/37#discussion_r271929743 posted by @nigelmegitt in https://github.com/w3c/PWETF/timeline_ I would like to remove the example "Well actually" because:

* It is based on an idiomatic understanding held by a subset of English speakers, whereas a different subset does not share the same understanding. I checked this by asking a couple of native English speakers in London - one did expect a sentence beginning "well actually" to be likely to include a negative statement, the other did not. 
* It is entirely possible to make a useful contribution to a conversation in a non-patronising way beginning with "well actually".
* It overly promotes one phrase as being a signal when many other constructions would have the same negative effect. In that sense it detracts from the intent of the statement it is supposed to be exemplifying, rather than adding to it.

The linked thread above ended with a common understanding that the phrase is indeed problematic for some people, and I do not intend to downplay that, however I still consider it a poor example to use, and as such propose that it be removed.

Consider elaborating this example with a sentence such as:

> Be aware that, regardless of the speaker's intentions, some phrases or constructions lead people to expect a patronising statement to follow, and avoid such phrases. For example, beginning an interjection with "Well actually..." can set this expectation, and be taken as a sign of disrespect.

### PR DESCRIPTION
**Pull request by chaals**
fixes #45 (at least, that's the intention).

Clarify that chairs and other in "leadership positions":
1. Are meant to help follow up an issue
2. Should be proactive in doing so
3. Are expected to take immediate proportionate action as necessary to stop a harmful situation from continuing.

***
<a href="https://pr-preview.s3.amazonaws.com/chaals/PWETF/pull/75.html" title="Last updated on Dec 19, 2019, 5:32 PM UTC (b659a04)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/PWETF/75/60544e7...chaals:b659a04.html" title="Last updated on Dec 19, 2019, 5:32 PM UTC (b659a04)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -216,9 +216,9 @@ <h3 id="unacceptablebehaviour">
       <li>Retaliating against anyone who files a complaint that someone has violated this code of conduct.</li>
     </ul>
 
-    <p>This Code prioritises the safety of individuals within marginalised communities over the
+    <p>The enforcers of this Code should prioritise the safety of individuals within marginalised communities over the
```
Comment (TzviyaSiegman) [inline index.html]: I believe @AdaRoseCannon was asked to change this to "this code"

File: index.html [hunk 2]
```diff
@@ -293,29 +293,56 @@ <h3>Expected Behavior</h3>
...
     <h2>Reporting</h2>
-    <p>If you are concerned about your immediate safety, contact <a href="https://en.wikipedia.org/wiki/List_of_emergency_telephone_numbers">local law enforcement</a>.
-      For a face to face event you may need to contact venue staff for assistance contacting them.</p>
-    <p>You may also contact a W3C ombud who will assist you: <a href="mailto:ombuds@w3.org">ombuds@w3.org</a>.</p>
+    <p>If you are concerned about your immediate safety, contact 
+      <a href="https://en.wikipedia.org/wiki/List_of_emergency_telephone_numbers">local law enforcement</a>.
+      For a face to face event Chairs and venue staff should provide any necessary assistance contacting them.
+    </p>
+    
+    <p>You may also contact a W3C ombud who will assist you. COntact addresses are included in the
```
Comment (TzviyaSiegman) [inline index.html]: s/COntact/Contact

File: index.html (hunk 1 above)
Comment (chaals) [reply index.html]: We each did in different branches. Anyway, merged now separate from this PR.

File: index.html (hunk 2 above)
Comment (chaals) [reply index.html]: Fixed in the PR

File: index.html (hunk 1 above)
Comment (vlevantovsky) [reply index.html]: It doesn't make sense to say that "the Code" ... "reserves the rights not to take further actions". It used to be "the enforcers" who are capable of making decisions - the Code prescribes recommended behavior but doesn't make decision decisions on its own. Rewrite needed.

Comment (chaals) [reply index.html]: Agreed. Noting that we need to do more work based on #74 ...

File: index.html [hunk 3]
```diff
@@ -293,29 +293,56 @@ <h3>Expected Behavior</h3>
     
   <section id="Reporting">
     <h2>Reporting</h2>
-    <p>If you are concerned about your immediate safety, contact <a href="https://en.wikipedia.org/wiki/List_of_emergency_telephone_numbers">local law enforcement</a>.
-      For a face to face event you may need to contact venue staff for assistance contacting them.</p>
-    <p>You may also contact a W3C ombud who will assist you: <a href="mailto:ombuds@w3.org">ombuds@w3.org</a>.</p>
+    <p>If you are concerned about your immediate safety, contact 
+      <a href="https://en.wikipedia.org/wiki/List_of_emergency_telephone_numbers">local law enforcement</a>.
+      For a face to face event Chairs and venue staff should provide any necessary assistance contacting them.
```
Comment (nigelmegitt) [inline index.html]: As a Chair I am only occasionally the _host_ of a meeting. If I am in a foreign country I feel no more able to assist with contacting law enforcement than anyone else. Propose that we introduce the concept of "host" for face to face meetings and expect them to help here, as well as venue staff.

Editorial nit: it took me three goes reading this to realise that the "them" at the end of the sentence refers to "local law enforcement" and not to "Chairs and venue staff". Perhaps there's a way to edit this so it is easier to parse. I don't have any immediate suggestions though :-(

File: index.html [hunk 4]
```diff
@@ -293,29 +293,56 @@ <h3>Expected Behavior</h3>
...
       the issue with an Ombuds person. You may also ask for them to help you raise it to an Ombuds person immediately
       if you do not feel comfortable contacting the Ombuds directly. 
     </p>
 
-    <p>If you are responsible for a community within the W3C such as a chair of a working group and you witness harassment or
-      any other behaviour which goes against this code you are encouraged to raise issues directly with
-      an <a href="mailto:ombuds@w3.org">Ombuds</a> person who can assist you.
+    <p>If you are in a <a>leadership position</a> and you witness harassment or other 
+      <a href="#unacceptablebehavior">unacceptable behaviour</a> you should raise issues directly with
+      an <a href="https://www.w3.org/Consortium/pwe/#ombuds">Ombuds</a> person who can assist you.
```
Comment (nigelmegitt) [inline index.html]: separating "Ombuds" from "person" with a space seems weird. Shouldn't it be "Ombud" or "Ombudsperson" with no space?

File: index.html [hunk 5]
```diff
@@ -293,29 +293,56 @@ <h3>Expected Behavior</h3>
...
     </p>
-
+    
+    <p>Any person in a <a>leadership position</a> should provide immediate assistance to any participant
+      who wants to contact an ombuds. Failure to do so may increase the harmful consequences of the action being reported.
```
Comment (nigelmegitt) [inline index.html]: s/ombuds/[whatever generic term we use for a single Ombudsperson]

File: index.html [hunk 6]
```diff
@@ -293,29 +293,56 @@ <h3>Expected Behavior</h3>
...
+    </p>
+    
+    <p>Chairs, Team Contacts and Event Organisers should take such immediate action as they deem necessary in order to stop
+      harmful effects of harassment, bullying, or other <a href="#unacceptablebehaviour">unacceptable behaviour</a>.
+      This action may take many forms, but examples include:
+    </p>
+    
+    <ul>
+      <li>ejecting someone from a meeting or event</li>
+      <li>Blocking them on a mailing list</li>
```
Comment (nigelmegitt) [inline index.html]: Consider adding:

`<li>Asking people temporarily to move to different rooms or locations</li>`

**Comment by TzviyaSiegman** [comment]
Note that at 12/12 meeting @AdaRoseCannon and I agreed to edit this PR. @nigelmegitt's comments do not reflect the discussion held at the 12/12 meeting.

**Comment by nigelmegitt** [comment]
@TzviyaSiegman thanks for drawing my attention to the fact this was discussed. I was unable to be present at that meeting due to a clash with another W3C meeting and was unaware of the discussion. My comments were not intended to be a reflection of the discussion of the meeting. It was purely a coincidence that I managed to get around to looking at this pull request at the time I did.

At the same time I hope you are not suggesting that my absence from the verbal discussion in the meeting somehow reduces the value of my contributions.

**Comment by TzviyaSiegman** [comment]
Thanks @nigelmegitt. Sorry you had to miss the meeting. @AdaRoseCannon and I will be editing the whole section. We will take your comments into account as we edit it. I did not mean to sound dismissive.

**Comment by TzviyaSiegman** [comment]
Thanks @nigelmegitt. Sorry you had to miss the meeting. @AdaRoseCannon and I will be editing the whole section. We will take your comments into account as we edit it. I did not mean to sound dismissive.

File: index.html (hunk 5 above)
Comment (AdaRoseCannon) [reply index.html]: ombuds is both a singular and plural noun. (it's short for ombudsman/ombudsmen)

File: index.html (hunk 3 above)
Comment (TzviyaSiegman) [reply index.html]: noted, we will pay attention to this as we work on chairs training. And, we will work on antecedents.

File: index.html (hunk 4 above)
Comment (TzviyaSiegman) [reply index.html]: this is a typo

File: index.html (hunk 5 above)
Comment (nigelmegitt) [reply index.html]: If we want to use it like that then we need to define it, because "ombud" is not an entry at all in the Oxford English Dictionary, so I would assume it is not universally understood.

File: index.html [hunk 7]
```diff
@@ -295,10 +295,10 @@ <h3>Expected Behavior</h3>
     <h2>Reporting</h2>
     <p>If you are concerned about your immediate safety, contact 
       <a href="https://en.wikipedia.org/wiki/List_of_emergency_telephone_numbers">local law enforcement</a>.
-      For a face to face event Chairs and venue staff should provide any necessary assistance contacting them.
+      For a face to face event Chairs and venue staff should provide any necessary assistance contacting law enforcement.
     </p>
     
-    <p>You may also contact a W3C ombud who will assist you. Contact addresses are included in the
+    <p>You may also contact a W3C ombuds who will assist you. Contact addresses are included in the
```
Comment (chaals) [inline index.html]: Having looked again today, I think we should use "ombudsperson" rather than just "ombuds".

File: index.html [hunk 8]
```diff
@@ -311,28 +311,33 @@ <h2>Reporting</h2>
...
+      <li>Point out if someone is violating the CEPC, to give them the chance to withdraw or edit their statement.</li>
+      <li>Remind participants that meetings & work operate under the CEPC.</li>
+      <li>Asking someone to leave a meeting, or a conversation thread.</li>
+    </ul>
+    <h3>After the meeting:</h3>
     <ul>
-      <li>ejecting someone from a meeting or event</li>
-      <li>Blocking them on a mailing list</li>
-      <li>Removing specific comments from an archive, or the harmful parts of such comments</li>
+      <li>Follow up with affected participants, this may be seperate meetings.</li>
```
Comment (nigelmegitt) [inline index.html]: spelling nit: s/seperate/separate

File: index.html [hunk 9]
```diff
@@ -311,28 +311,33 @@ <h2>Reporting</h2>
...
+      <li>The Chair may reach out to an Ombuds for assistance</li>
+      <li>If the Chair needs assistance, they can make use of the Chairs Training program.</li>
```
Comment (nigelmegitt) [inline index.html]: If I need assistance in this situation I would not want to go to a training programme, I would want to go to a person. Suggest rewording to:

>Further information and resources for Chairs are available via the Chairs Training program.

Comment (chaals) [reply index.html]: agree - and move it out of the list to be a following paragraph

**Review by Jeffkeen7 — COMMENTED** [review]
Jeffrey Martin Keen owner and Creator
